### PR TITLE
Add and Update Rail Diagrams

### PR DESCRIPTION
**Pull request by SquareScreamYT**

### CODE REVIEW

File: src/main/java/org/nguh/nguhcraft/NguhPaintings.kt [hunk 1]
```diff
@@ -17,9 +17,10 @@ object NguhPaintings {
     val MAP = of("map")
     val RABO = of("rabo")
     val RAIL_DIAGRAM = of("rail_diagram")
+    val CHILLVANA_METRO = of("chillvana_metro")
```
Comment (Sirraide) [inline src/main/java/org/nguh/nguhcraft/NguhPaintings.kt:20]: On that note, these are also supposed to be sorted

Comment (SquareScreamYT) [reply src/main/java/org/nguh/nguhcraft/NguhPaintings.kt:20]: alright

**Comment by SquareScreamYT** [comment]
Fixed

**Comment by Sirraide** [comment]
The rail diagram image seems to have black letterboxing at the top and bottom; those were probably supposed to be white pixels instead.

image

**Comment by Sirraide** [comment]
(the other images seem fine tho)

**Comment by SquareScreamYT** [comment]
Fixed

**Review by Sirraide — APPROVED** [review]
Alright, everything seems fine now

**Pull request merged**